src/main/java/com/itschool/eventmanagment/config/LocalDateTimeConverter.java

```diff
@@ -0,0 +1,22 @@
+package com.itschool.eventmanagment.config;
+
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+@Component
+public class LocalDateTimeConverter implements Converter<String, LocalDateTime> {
```

Comment:
We don't need this component since we can use Spring's built-in date parsing and can automatically handle `LocalDateTime` parameters using `@DateTimeFormat`. You can annotate controller's method parameter:

`@RequestParam(required = false) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime from`